crates/ty_python_semantic/resources/mdtest/function/return_type.md

```diff
@@ -455,7 +455,7 @@ def i() -> typing.Iterable:
     yield 42
 
 def i2() -> typing.Generator:
-    yield from i()
+    yield from i()  # error: [not-iterable]
```

Comment:
??

Reply:
Ah, this happens because `i` is redefined below. This test should probably use subsections.